test/integration_test/backup_test.go

```diff
@@ -0,0 +1,137 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"fmt"
+	"testing"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	"github.com/portworx/sched-ops/k8s"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	instanceID = "basic"
+	appKey     = "mysql-app-to-backup"
+)
+
+func testBackup(t *testing.T) {
+	t.Run("backupTest", backupRestoreDeleteTest)
+}
+
+var backupLocationMap = map[string]storkv1.BackupLocationType{
```

Comment:
You don't need this, you can pass in `storkv1.BackupLocationType` to `createBackupLocation()`